### PR TITLE
Map Control Scrolling/Focus

### PR DESCRIPTION
**Pull request by maatvirtue**
I added scrolling / focus to the map control.

### CODE REVIEW

File: src/main/java/net/forgottenschism/gui/bean/Position2d.java [hunk 1]
```diff
@@ -18,8 +26,58 @@ public Position2d(int x, int y)
...
+	public boolean smallerThan(Position2d position)
+	{
+		return x<position.getX() && y<position.getY();
+	}
+
+	/**
+	 * A position is greater than or equal to another position if its X and Y coordinates are greater than or equal to the
+	 * other position's.
+	 */
+	public boolean greaterOrEqualThan(Position2d position)
```
Comment (TenaciousDan) [inline src/main/java/net/forgottenschism/gui/bean/Position2d.java]: This method should be called 'greaterThanOrEqualTo'. It just sounds better...

File: src/main/java/net/forgottenschism/gui/bean/Position2d.java [hunk 2]
```diff
@@ -18,8 +26,58 @@ public Position2d(int x, int y)
...
+	{
+		return x>=position.getX() && y>=position.getY();
+	}
+
+	/**
+	 * A position is smaller than or equal to another position if its X and Y coordinates are smaller than or equal to the
+	 * other position's.
+	 */
+	public boolean smallerOrEqualThan(Position2d position)
```
Comment (TenaciousDan) [inline src/main/java/net/forgottenschism/gui/bean/Position2d.java]: LessThanOrEqualTo?

File: src/main/java/net/forgottenschism/gui/bean/Position2d.java (hunk 1 above)
Comment (maatvirtue) [inline src/main/java/net/forgottenschism/gui/bean/Position2d.java]: I agree

File: src/main/java/net/forgottenschism/gui/bean/Position2d.java (hunk 2 above)
Comment (maatvirtue) [inline src/main/java/net/forgottenschism/gui/bean/Position2d.java]: ok

**Comment by TenaciousDan** [comment]
It looks good. Although, and i dont want to be a stickler, but you changed the word 'smaller' to 'less' in one method name but not the other. Other than that I think it's ready to merge with master.

File: src/main/java/net/forgottenschism/gui/bean/Position2d.java [hunk 3]
```diff
@@ -18,8 +26,58 @@ public Position2d(int x, int y)
...
+	 */
+	public boolean greaterThan(Position2d position)
+	{
+		return x>position.getX() && y>position.getY();
+	}
+
+	/**
+	 * A position is smaller than another position if its X and Y coordinates are smaller.
+	 */
+	public boolean smallerThan(Position2d position)
```
Comment (TenaciousDan) [inline src/main/java/net/forgottenschism/gui/bean/Position2d.java]: This should also be called 'lessThan' to keep it consistent...

**Comment by TenaciousDan** [comment]
It's good to go! Merge away!!!

**Pull request merged**